Apsitvarkom.Models/Public/TranslatedResponse.cs

```diff
@@ -0,0 +1,18 @@
+﻿using FluentValidation;
+
+namespace Apsitvarkom.Models.Public;
+
+public class TranslatedResponse<T>
+{
+    public T en { get; set; } = default!;
+    public T lt { get; set; } = default!;
```

Comment:
Having these in PascalCase will still return them as camelCase, so maybe it's worth keeping consistency and change these to PascalCase?